src/Symfony/Component/Notifier/Bridge/Smsmode/SmsmodeOptions.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Notifier\Bridge\Smsmode;
+
+use Symfony\Component\Notifier\Message\MessageOptionsInterface;
+
+/**
+ * @author gnito-org <https://github.com/gnito-org>
+ */
+final class SmsmodeOptions implements MessageOptionsInterface
```

Comment:
Option classes should have a unit case, maybe you can add this on your todo-list for the already merged bridges, that would be awesome 🙏 thanks